frontend/src/components/bucket/BucketSidebar.vue

```diff
@@ -13,10 +18,37 @@ const props = withDefaults(defineProps<Props>(), {});
 // Emits
 const emit = defineEmits(['close-sidebar-info']);
 
+// Store
+const permissionStore = usePermissionStore();
+const userStore = useUserStore();
+
+const { getBucketPermissions } = storeToRefs(permissionStore);
+const { userSearch } = storeToRefs(userStore);
+
+// State
+const managedBy: Ref<string | undefined> = ref();
+
 // Actions
 const closeSidebarInfo = async () => {
   emit('close-sidebar-info');
 };
+
+async function load() {
+  await permissionStore.fetchBucketPermissions({bucketId: props.sidebarInfo.bucketId});
+
+  const uniqueIds = [...new Set(getBucketPermissions.value
+    .filter( (x: BucketPermission) => x.bucketId === props.sidebarInfo.bucketId && x.permCode === Permissions.MANAGE )
+    .map( (x: BucketPermission) => x.userId) )];
+
+  if(uniqueIds.length) {
+    await userStore.fetchUsers({userId: uniqueIds} );
+    managedBy.value = userSearch.value.map( x => x.fullName ).join( ', ');
+  }
+}
+
+onMounted(() => {
+  load();
+});
```

Comment:
We should strongly consider using an earlier Vue lifecycle hook for this. I would recommend we use the `onBeforeMounted` event hook instead of `onMounted`. As well, since load is only ever used in one place, that can be consolidated to be the body of the `onBeforeMounted` callback function instead.

```ts
onBeforeMounted(() => {
  await permissionStore.fetchBucketPermissions({bucketId: props.sidebarInfo.bucketId});
  const uniqueIds = [...new Set(getBucketPermissions.value
    .filter( (x: BucketPermission) => x.bucketId === props.sidebarInfo.bucketId && x.permCode === Permissions.MANAGE )
    .map( (x: BucketPermission) => x.userId) )];
  if(uniqueIds.length) {
    await userStore.fetchUsers({userId: uniqueIds} );
    managedBy.value = userSearch.value.map( x => x.fullName ).join(', ');
  }
});